hack/lib/golang.sh

```diff
@@ -680,7 +680,7 @@ kube::golang::build_binaries() {
     # Use eval to preserve embedded quoted strings.
     local goflags goldflags goasmflags gogcflags
     eval "goflags=(${GOFLAGS:-})"
-    goldflags="${GOLDFLAGS:-} -s -w $(kube::version::ldflags)"
+    goldflags="${GOLDFLAGS:-} $(kube::version::ldflags)"
```

Comment:
So we added this in https://github.com/kubernetes/kubernetes/pull/69003..

Is reproducible only really critical for a release?

Is it better to set something like `${GOLDFLAGS:-'-w -s'}` here and have it as an overridable default, or would we WANT to hammer GOLDFLAGS at release build time?

WDYT, @dims? It doesn't look like we're currently doing any GOLDFLAGS changes at release time right now, other than `kube::version::ldflags`, which we're accounting for.

Reply:
I like that suggestion @cblecker , just so we don't have any surprises in the release builds. 👍